packages/fxa-payments-server/src/components/NewUserEmailForm/index.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../../../fxa-content-server/app/styles/variables';
+
+.new-user-email-form {
+    position: relative;
+
+    .sign-in-copy {
```

Comment:
It looks like the "Sign in" link should be black and underlined per the [final designs](https://www.figma.com/file/cOIAuHIIbrPbAlAIcpejNM/%F0%9F%8E%A1MVP-Subscription-Optimization-Flow?node-id=315%3A12945).